utils/FilePath.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace WeatherMonitoringSystem.utils
+{
+    public class FilePath
+    {
+        public static string filePath = @"D:\backend_training\C#\WeatherMonitoringSystem\config.json";
```

Comment:
You should use a relative path and copy the configuration file to the output directory. That way you're not depending on a path on your physical machine.

In `.csproj` you can do the following:

```xml
	<ItemGroup>
		<None Update="config.json">
			<CopyToOutputDirectory>Always</CopyToOutputDirectory>
		</None>
	</ItemGroup>
```

That way, the build would always copy `config.json` to `./bin/Debug/net8.0/` and hence it would be directly visible to your code as `config.json`.

If you're going to use the configuration builder then it mentions how to do these steps.